File: code-alongs/04-ruby/99. warmups/even_multiples.rb

def even_multiples(list_of_numbers)
    evens = list_of_numbers.select{ |num| num.even? || num%5 == 0 }  
    evens.sort
end

print even_multiples([14, 5, 7, 2, 25, 10, -4, 3, 3, 25, 2, 6, 5])

File: code-alongs/08-ajax/99. warmups/even_multiples.js
function evenMultiples(listOfNums) {
    return listOfNums.filter(num => num % 2 == 0 || num % 5 == 0)
}

console.log(evenMultiples([14, 5, 7, 2, 25, 10, -4, 3, 3, 25, 2, 6, 5]))